app/views/posts/show.html.erb

```diff
@@ -4,6 +4,9 @@
   <div class="com_comments_container">
     <% @post.comments.each do |comment| %>
       <%= render partial: 'shared/comment', locals: { comment: comment } %>
+      <% if can? :delete, comment %>
+      <button type="button" class="delete_btn btn"><%= link_to "Delete Comment", user_post_comment_path(user_id: comment.author.id, post_id: comment.post.id, id: comment.id), data: { turbo_method: :delete } %></button>
```

Comment:
- [ ] An error occurs on the post's show page so I'm unable to check if only authorized users can delete a post
- [ ] Due to the aforementioned error, it is also impossible to check if comments can only be deleted by their owners and admin users. Please try to fix the error shown below 👇 
![image](https://user-images.githubusercontent.com/46839548/218226050-fa1917d1-1de6-44a3-b8d1-cd279388627f.png)